nbgrader/converters/base.py

```diff
@@ -359,3 +363,28 @@ def _handle_failure(gd):
 
             self.log.error(msg)
             raise NbGraderException(msg)
+
+    def recursively_archive(self, directory, archive_name):
+        all_files_and_folders = [ item for item in os.listdir(directory) if item != archive_name+".zip" ]
```

Comment:
I know this is a little nitpicky, but I'd appreciate if you could match your code style to the same style as in the rest of the file, which largely follows PEP8---i.e. spaces around `+` signs, no spaces after `[` or before `]`, etc.